ceph_iscsi_config/target.py

```diff
@@ -681,33 +681,37 @@ def manage(self, mode):
             else:
                 # create the target
                 self.create_target()
-                seed_target = {
-                    'disks': {},
-                    'clients': {},
-                    'acl_enabled': True,
-                    'auth': {
-                        'username': '',
-                        'password': '',
-                        'password_encryption_enabled': False,
-                        'mutual_username': '',
-                        'mutual_password': '',
-                        'mutual_password_encryption_enabled': False},
-                    'portals': {},
-                    'groups': {},
-                    'controls': {}
-                }
-                config.add_item("targets", self.iqn, seed_target)
-                config.commit()
-
-                discovery_auth_config = config.config['discovery_auth']
-                Discovery.set_discovery_auth_lio(discovery_auth_config['username'],
-                                                 discovery_auth_config['password'],
-                                                 discovery_auth_config[
-                                                     'password_encryption_enabled'],
-                                                 discovery_auth_config['mutual_username'],
-                                                 discovery_auth_config['mutual_password'],
-                                                 discovery_auth_config[
-                                                     'mutual_password_encryption_enabled'])
+                # if error happens, we should never store this target to config
+                if self.error:
+                    self.logger.error("Failed to create target")
```

Comment:
Nit: just `return` on error and then you can avoid the `else` branch and associated indentation